libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/SdkVersionChecker.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+class SdkVersionChecker(private val currentSdkVersion: Int) {
```

Comment:
what is an aim of this class? may be we can archive it by another way

Reply:
why is it public? it could be private, don't you think so? cc @Guardiola31337

Reply:
I think it's good to have this wrapper, cause it makes this kind of meta-code checks like a plain text, and increase readability of code. 
And yeah, it make sense to make it internal... but it also used in UI library... maybe we need to duplicate this one in UI module 🤦‍♂ (I know it is very bad... just suggestion)? cc @Guardiola31337

Reply:
lets keep it as it is. Keep moving with migration. May be we should create separate module with some utils and include it to each module through gradle's dependencies as internal